File: HealthSysHub.Web.DBConfiguration/Models/PharmacyOrderItem.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace HealthSysHub.Web.DBConfiguration.Models
{
    [Table("PharmacyOrderItem")]
    public class PharmacyOrderItem
    {
        [Key]
        public Guid PharmacyOrderItemId { get; set; }
        public Guid? PharmacyOrderId { get; set; }
        public Guid? PharmacyId { get; set; }
        public Guid? MedicineId { get; set; }
        public long? ItemQty { get; set; }
        public decimal? UnitPrice { get; set; }
        public decimal? TotalAmount { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset? CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/pharmacyorderitem.ts
export interface PharmacyOrderItem {
  pharmacyOrderItemId: string;
  pharmacyOrderId?: string | null;
  pharmacyId?: string | null;
  medicineId?: string | null;
  itemQty?: number | null;
  unitPrice?: number | null;
  totalAmount?: number | null;
  createdBy?: string | null;
  createdOn?: Date | string | null;
  modifiedBy?: string | null;
  modifiedOn?: Date | string | null;
  isActive: boolean;
}
